aspnet/tutorials/first-mvc-app/new-field.rst

```diff
@@ -1,23 +1,22 @@
 Adding a New Field
-==================================================
+================================================
 
 By `Rick Anderson`_
 
-In this section you'll use `Entity Framework <http://ef.readthedocs.io/en/latest/getting-started.html>`__ Code First Migrations to migrate some changes to the model classes so the change is applied to the database.
+In this section you'll use `Entity Framework <http://ef.readthedocs.io/en/latest/platforms/aspnetcore/new-db.html>`__ Code First Migrations to add a new field to the model and migrate that change to the database.
```

Comment:
> In this section you'll use `Entity Framework <http://ef.readthedocs.io/en/latest/platforms/aspnetcore/new-db.html>`__ 

In this section you'll use `` `Entity Framework <http://docs.efproject.net/en/latest/platforms/aspnetcore/new-db.html>`__ ``